anta/cli/exec/commands.py

```diff
@@ -43,12 +44,17 @@ def clear_counters(ctx: click.Context, log_level: str, tags: str) -> None:
         )
 
 
+def _get_snapshot_dir(ctx: click.Context, param: click.Parameter, value: str) -> str:  # pylint: disable=unused-argument
```

Comment:
Maybe just a basic docstring to help understand the goal of it

Reply:
Not in this PR anymore, see #166